### PR TITLE
Users/sgupta/maven preview

### PR DESCRIPTION
**Pull request by shashwatgupta**
This Pull request adds Duplicate of Maven task. This task would be under preview - We will remove it later with servicing steps. This has Maven task along with CodeCoverage Publishing tab

### CODE REVIEW

File: Tasks/MavenPreview/task.json [hunk 1]
```diff
@@ -0,0 +1,170 @@
+{
+    "id": "B8C7DDE3-ACA0-4D27-B1EF-259806BC31B6",
+    "name": "MavenCC",
+    "friendlyName": "Maven With CodeCoverage",
+    "description": "Build with Apache Maven and check CodeCoverage",
+    "helpMarkDown": "[More Information](http://go.microsoft.com/fwlink/?LinkID=613723)",
+    "category": "Build",
+    "visibility": [
+                "Preview",
```
Comment (acesiddhu) [inline Tasks/MavenPreview/task.json]: wont this be available for RM flows as well

File: Tasks/MavenPreview/task.json [hunk 2]
```diff
@@ -0,0 +1,170 @@
+{
+    "id": "B8C7DDE3-ACA0-4D27-B1EF-259806BC31B6",
```
Comment (acesiddhu) [inline Tasks/MavenPreview/task.json]: hope this is a new guid

**Comment by acesiddhu** [comment]
rest looks fine

File: Tasks/MavenPreview/task.json (hunk 1 above)
Comment (shashwatgupta) [inline Tasks/MavenPreview/task.json]: Maven task itself is not avilble in RM Flow

File: Tasks/MavenPreview/task.json (hunk 2 above)
Comment (shashwatgupta) [inline Tasks/MavenPreview/task.json]: yaa its new GUID

**Pull request merged**